mex/extractors/synopse/main.py

```diff
@@ -259,10 +259,10 @@ def synopse_activity() -> dict[str, Any]:
 
 
 @asset(group_name="synopse")
-def extracted_synopse_activities(  # noqa: PLR0913
+def synopse_activities(  # noqa: PLR0913
```

Comment:
if the variable asset is called `synopse_extracted_variables` why not `synopse_extracted_activities`?